### ISSUE TITLE
Кастомное поле заголовка

### ISSUE THREAD

**Issue by nanomen** [av3nger/market-exporter#33]
Добрый день, столкнулись с проблемой проверки xml файла.

Яндекс маркет требует заголовки в определенном формате: тип или категория товара + производитель или бренд + модель товара и важные параметры (Кроссовки Adidas (VLNEO) V RACER NYLON)

А сейчас в заголовках передается только название товара.

Подскажите, как правильнее решить проблему? Не хочется исходный код Вашего плагина трогать.
Спасибо!

**Comment by av3nger** [comment]
Добрый день, а можете показать это требование?

**Comment by nanomen** [comment]
Конечно, вот скрин сделал https://yadi.sk/i/l5eTNbKY3G68ZV, а вот ссылка на документацию https://yandex.ru/support/partnermarket/registration/pricelist.html. Пункт "Корректно составьте наименования товарных предложений"

**Comment by av3nger** [comment]
Я не знал. Спасибо. Поправлю на днях

**Comment by nanomen** [comment]
О, это было бы прекрасно, буду очень ждать, спасибо!
